README.md

```diff
@@ -170,12 +171,21 @@ Immediately triggers a new search for the given text
 
 - text `String` A word of sequence of words to search for
 
-#### `search.playResult(result)`
+**Returns:**
+- retVal `Promise` - A promise that will resolve with the results of the search you just performed.
+The promise will be rejected with a timeout error if the search takes too long.
+
+#### `search.playResult(searchText, result)`
 Immediately attempts to play the given result.  If we fail to play the given result
 an error will be thrown.
 
+- searchText `String` The search text that causes the search result to appear
```

Comment:
Adding a requisite for `searchText` feels a bit strange since we will probably double search if someone has already done a search. I think 2 function calls isn't too bad (3 was pretty bad):

```js
var p = search.performSearch('my text');
p.resolve(function (results) {
  search.playResult(results.songs[0]);
});
```